package.json

```diff
@@ -93,8 +93,10 @@
     "chromedriver": "^83.0.1",
     "css-loader": "^3.4.2",
     "eslint": "^6.8.0",
+    "eslint-config-prettier": "^6.11.0",
     "eslint-plugin-flowtype": "^4.6.0",
     "eslint-plugin-jest": "^23.8.2",
+    "eslint-plugin-prettier": "^3.1.4",
```

Comment:
I might be missing something, but since you're running `&& prettier . -c` on `"lint"` aren't both eslint-config/plugin-prettier unnecessary?

Reply:
I'll double check this, but I was initially not seeing any errors when I ran `lint`. But at the same time I was doing a lot of swapping between branches so might've been looking at the wrong thing.

Reply:
OK, definitely not required. Removing it.